pipelines/tikv/pd/latest/pod-ghpr_build.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    fsGroup: 1000
+  containers:
+    - name: golang
+      image: "hub.pingcap.net/jenkins/centos7_golang-1.18:latest"
```

Comment:
need use go1.19 image 'hub.pingcap.net/jenkins/centos7_golang-1.19:latest'

Reply:
old is still go-1.18

Reply:
fixed